### PR TITLE
 Changing install signatures from yeoman-generator@3.0.0

### PR DESCRIPTION
**Pull request by ErikElkins**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes:  https://github.com/yeoman/generator/commit/240a666ec84fcbcfe80897ae96aa80f908d4a368#diff-c9a43560dd6d606dc7759d0eda3fb582
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/yeoman-generator/index.d.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ declare class Generator extends EventEmitter {
      * @param spawnOptions Options to pass `child_process.spawn`.
      * @return Resolved if install successful, rejected otherwise
```
Comment (ikatyang) [inline types/yeoman-generator/index.d.ts]: Could you update these JSDoc as well?

**Review by ikatyang — CHANGES_REQUESTED** [review]

**Review by janslow — COMMENTED** [review]
LGTM, once @ikatyang's comments are resolved

File: types/yeoman-generator/index.d.ts (hunk 1 above)
Comment (CaringIsCreepy) [reply types/yeoman-generator/index.d.ts]: Updated

**Review by ikatyang — APPROVED** [review]

**Review by janslow — APPROVED** [review]

**Comment by CaringIsCreepy** [comment]
@janslow i just fixed linting ... should be good to go.

**Review by janslow — APPROVED** [review]

**Pull request merged**